client/src/main/java/com/alibaba/nacos/client/naming/core/HostReactor.java

```diff
@@ -304,6 +304,7 @@ public void refreshOnly(String serviceName, String clusters) {
 
     public class UpdateTask implements Runnable {
         long lastRefTime = Long.MAX_VALUE;
+        long delayTime = -1;
```

Comment:
I don't think this should be a member property, just a local variable for a method

Reply:
Yes,Your suggestion is very good, `delayTime` more suitable as a method local variable